sklearn/decomposition/_nmf.py

```diff
@@ -1327,7 +1327,7 @@ def inverse_transform(self, Xt=None, W=None):
 
         Returns
         -------
-        X : {ndarray, sparse matrix} of shape (n_samples, n_features)
+        X : ndarray of shape (n_samples, n_features)
```

Comment:
Indeed, this is in line with the docstring of `_fit_transform` that should always return `H` as numpy arrays for both solvers (coordinate descent and multiplicative update).